src/components/HeaderNav.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+const HeaderNav = () => {
+    return(
+        <nav>
+            <ul>
```

Comment:
I think this is great for now, but we will want 'About' and 'Contact' to be anchor tags (wrapped by list items). 'Add Credits' and our 'Cart' will be buttons (also wrapped in `li` elements).